src/theme.js

```diff
@@ -29,22 +37,60 @@ export const grommetToolsTheme = {
            "Apple Color Emoji", 
            "Segoe UI Emoji", 
            "Segoe UI Symbol"`,
+    },
+  },
+  anchor: {
+    color: 'black',
+  },
+  box: {
+    size: {
+      headerSize: '36px',
+      xlarge: '200px',
+    },
+  },
+  textInput: {
+    input: {
+      font: {
+        size: '18px',
+      },
+    },
+  },
+  formField: {
+    border: {
+      position: null,
+    },
+  },
+  icon: {
+    size: {
+      small: '36px',
+      large: '60px',
+    },
+  },
+  heading: {
```

Comment:
I do not think the heading should be overriden

Reply:
I changed heading so that in the theme I am not overriding values and instead creating a 'customMedium' and 'customLarge' for the Grommet Tools and Grommet Tool Suite headings

Reply:
we shouldn't override the grommet theme and use it as is.